components/SettingsControls/src/SettingsCard/SettingsCard.cs

```diff
@@ -21,6 +21,9 @@ namespace CommunityToolkit.WinUI.Controls;
 [TemplateVisualState(Name = PressedState, GroupName = CommonStates)]
 [TemplateVisualState(Name = DisabledState, GroupName = CommonStates)]
 
+[TemplateVisualState(Name = BitmapHeaderIconEnabledState, GroupName = CommonStates)]
+[TemplateVisualState(Name = BitmapHeaderIconDisabledState, GroupName = CommonStates)]
```

Comment:
These should belong to "BitmapHeaderIconStates" instead of "CommonStates".
I'm a bit surprised that works anyway though. 😅